docs/blog/_posts/2021-03-31-scala3-rc2.md

```diff
@@ -0,0 +1,92 @@
+
+Read more about this release below.
+
+<!--more-->
+# Significance and further steps
+The 3.0.0-RC releases are release candidates for the 3.0.0 final. This means we aim to pack all the features meant for 3.0.0 into these releases and ensure their stability. After a grace period during which the community tests the latest release candidate, if no major problems are discovered, that release candidate becomes 3.0.0 stable.
+
+3.0.0-RC2 is the next iteration of this release model. Its primary objective is to get tested by the community and generate feedback. If no major issues are discovered, we are planning to release 3.0.0 stable in 4 weeks from now. If there are indeed major issues discovered with RC2, we will have to release RC3 in between – possibly earlier than 4 weeks from now to speed up the way to 3.0.0.
+
+Only critical issue fixes will find its way into 3.0.0 from now on, and any such fix will mean RC3. In the meanwhile, all the non-critical commits will appear only in 3.0.1. The way it will work is that we have a separate branch for 3.0.0, and we are going to backport the critical issue fixes into that branch from the main, 3.0.1, branch.
+
+Post-3.0.0, we are planning to proceed with the release cycle we have had before 3.0.0. That is, we bump the stable patch version every 6 weeks and make an RC for that version. Simultaneously we make the RC of the previous patch version into a stable version.
+
+Concretely, this means in 6 weeks from now we'll have 3.0.1-RC1, and in 12 weeks, we'll promote 3.0.1-RC1 into 3.0.1 stable and release 3.0.2-RC1.
+
+# Changes for 3.0.0-RC2
+As mentioned above, we are currently in an issue-fixing mode. So a lot of those 200+ PRs for this release were about issue fixing.
+
+There are some notable changes worth mentioning.
+
+## Restrict experimental features to unstable releases only
+PR [#11920](https://github.com/lampepfl/dotty/pull/11920) restricts usage of experimental features only to nightlies and snapshots. This change ensures that changes deemed experimental will not propagate into the wider ecosystem provided that wider ecosystem depends on stable releases. This is needed so that if an experimental feature is removed from the language, the ecosystem will not be impacted.
+
+## New `unsafeNulls` language feature
+PR [#9884](https://github.com/lampepfl/dotty/pull/9884) adds a new language feature which enables explicit nulls checks. From now on, you can use an import `import scala.language.unsafeNulls` to enable the feature, no compiler flags needed. For discussion, see the PR linked above, and for more information on the feature, see the [documentation](https://dotty.epfl.ch/docs/reference/other-new-features/explicit-nulls.html).
+
+## Treat Scala.js pseudo-unions as real unions
+In PR [#11671](https://github.com/lampepfl/dotty/pull/11671), we now treat the `scala.scalajs.js.|[A, B]` as if it was a real Scala 3 union `A | B`, which further boosts the support for Scala.js in Scala 3.
+
```

Comment:
should have the change to enable the initialisation checker from `-Ycheck-init` to `-Ysafe-init`